hack/e2e-suite/guestbook.sh

```diff
@@ -38,15 +38,22 @@ GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
 function teardown() {
   ${KUBECTL} stop -f "${GUESTBOOK}"
   if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
-    gcloud compute forwarding-rules delete "${INSTANCE_PREFIX}-default-frontend" || true
-    gcloud compute target-pools delete "${INSTANCE_PREFIX}-default-frontend" || true
-  fi  
+    gcloud compute forwarding-rules delete -q "${INSTANCE_PREFIX}-default-frontend" || true
+    gcloud compute target-pools delete -q "${INSTANCE_PREFIX}-default-frontend" || true
+  fi
 }
 
 prepare-e2e
 
 trap "teardown" EXIT
 
+if gcloud config list compute/region --format text | grep 'compute.region: None'; then
```

Comment:
I think you need to hack out region instead, see e.g. https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gce/util.sh#L625.